### DISCUSSION TITLE
GTM slows down Lighthouse score

### DISCUSSION THREAD

**Discussion by tomytubert** [vercel/next.js#50032] [Help]
### Summary

Scripts are added in _document.js 
I've added strategy="afterInteractive" to the <Script>.
I don't know what else I can do to improve the score.

Example of how I add scripts:
` <Script
          strategy="afterInteractive"
          src="https://www.googletagmanager.com/gtag/js?id=GMID"
        ></Script>`

Here is my lighthouse analysis: https://pagespeed.web.dev/analysis/https-ohmydog-io/0f93348iev?form_factor=mobile

**Comment by noob-kittu** [comment]
same issue i'm facing with mobile speed of my website https://enhanceai.art/
